app/Http/Controllers/EntryController.php

```diff
@@ -144,9 +144,28 @@ public function update(EntryUpdateRequest $request, Entry $entry)
      * @param \App\Models\Entry $entry
      * @return \Illuminate\Http\RedirectResponse
      */
-    public function destroy(Request $request, Entry $entry)
+    public function destroy(Request $request, Entry $entry = null)
     {
-        $entry->delete();
+        $user = $request->user();
+        $data = $request->validate([
+            'entries' => 'sometimes|array'
+        ]);
+
+        if ($request->has('entries')) {
+            $entries = Entry::whereIn('id', $data['entries'])->get();
+
+            foreach ($entries as $entry) {
```

Comment:
I would rename the **loop** `$entry` here to something slightly different just to avoid any confusion with the `$entry` **parameter** of the method. Maybe `$arrayEntry` just as an example.